python-threatexchange/threatexchange/api.py

```diff
@@ -285,6 +287,41 @@ def get_threat_updates(
         url = f"{self._base_url}/{privacy_group}/threat_updates/"
         return _CursoredResponse(self, url, params, decode_fn=decode_fn)
 
+    def get_threat_privacy_groups_member(
+        self,
+    ) -> t.List[ThreatPrivacyGroup]:
+        """
+        Returns a non-paginated list of all privacy groups the current app is a
+        member of.
+        """
+        url = self._get_graph_api_url(f"{self.app_id}/threat_privacy_groups_member")
+        response = self.get_json_from_url(url)
+        return [ThreatPrivacyGroup.from_graph_api_dict(d) for d in response["data"]]
+
+    def _get_graph_api_url(
+        self, sub_path: t.Optional[str], query_dict: t.Dict = {}
+    ) -> str:
+        """
+        Returns a threatexchange URL for a sub-path and a dictionary of query
+        parameters. Automatically adds access_token to the query dictionary.
+        """
+        if "access_token" not in query_dict:
```

Comment:
(ignorable): Hmm, I'm super apprehensive about a method that breaks the convention that an API instance only belongs to a single app (single token).

I'd rather unconditionally rewrite it every time

Reply:
I'm not seeing it, @Dcallies.

You'd rather I skip the 'if' and always overwrite the "access_token" value?

Reply:
This is a private method, so only internal users 'should' call it. I'll remove the `if` statement.